src/main/webapp/about.jsp

```diff
@@ -34,7 +34,20 @@
 
   <div id="container">
     <div
-      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px;">
+      style="width:75%; margin-left:auto; margin-right:auto; margin-top: 50px; font-size: 15px;">
```

Comment:
Please deploy any UI changes locally and attach a screenshot.  :)

Reply:
![ui_change_font15px](https://user-images.githubusercontent.com/36179870/37184159-b2f17572-22ee-11e8-8c34-de31ee44c93b.jpg)
![ui_change_original](https://user-images.githubusercontent.com/36179870/37184160-b30f9c28-22ee-11e8-8219-7217c5c5dc1a.jpg)

Reply:
I feel that the original font-size looks better, so I removed my changes for the font-size

Reply:
Looks good thanks :)